docs/static/resources/modsecurity-deployment-auditlog-sidecar.yaml

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    run: modsecurity-spoa
+  name: modsecurity-spoa
+  namespace: ingress-controller
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      run: modsecurity-spoa
+  template:
+    metadata:
+      labels:
+        run: modsecurity-spoa
+        spec:
+          volumes:
+          - name: varlog
+            emptyDir: {}
+          containers:
+          - name: modsecurity-spoa
+            image: quay.io/jcmoraisjr/modsecurity-spoa
+            args:
+            - -n
+            - "1"
+            ports:
+            - containerPort: 12345
+              hostPort: 12345
```

Comment:
We don't need `hostPort` anymore. Remember to remove from the `modsecurity-deployment.yaml` as well.